eras/allegra/impl/cardano-ledger-allegra.cabal

```
@@ -70,6 +70,7 @@ library
     bytestring,
     cardano-ledger-binary >=1.4,
     cardano-ledger-core >=1.17,
+    cardano-ledger-core:internal,
```

We've historically used this syntax instead:
```suggestion
    cardano-ledger-core:{cardano-ledger-core,internal} >=1.17,
```
Let's continue with it for consistency sake. Same for other affected cabal files in this PR